docs/dev-guide/service-preview.md

```diff
@@ -0,0 +1,169 @@
+# Service Preview
+
+**This page is deprecated. For the latest version, check out [Edge Control](../../reference/edge-control).**
+
+How do you verify that the code you've written actually works? Ambassador's *Service Preview* lets developers see exactly how their service works in a realistic enviroment -- without impacting other developers or end users. Service Preview integrates [Telepresence](https://www.telepresence.io), the popular CNCF project for local development and debugging on Kubernetes.
+
+## Install `apictl`
+
+`apictl` is the command client for Ambassador.
+
+Download the latest version of the client:
+
+<a class="apictl-dl" href="https://s3.amazonaws.com/datawire-static-files/apictl/$aproVersion$/darwin/amd64/apictl">Mac 64-bit</a> |
+<a class="apictl-linux-dl" href="https://s3.amazonaws.com/datawire-static-files/apictl/$aproVersion$/linux/amd64/apictl">Linux 64-bit</a>
+
+Make sure the client is somewhere on your PATH. In addition, place your license key in `~/.ambassador.key`.
+
+Information about open source code used in `apictl` can be found by running `apictl --version`.
+
+## Getting started
+
+In this quick start, we're going to preview a change we make to the backend service of the quote application, without impacting normal users of the application. Before getting started, make sure the [quote application is installed](../../../user-guide/install-ambassador-oss) on your cluster and you've installed the `apictl` command line tool, as explained above.
```

Comment:
```diff
- [quote application is installed](../../../user-guide/install-ambassador-oss)
+ [quote application is installed](../../../user-guide/install#create-a-mapping)
```

Reply:
making this link to getting started, since we'll be switching those pages shortly.